### DISCUSSION TITLE
Error: Could not find a valid build in the '/workspace/build' directory! Try building your app with 'next build' before starting the server.

### DISCUSSION THREAD

**Discussion by barbaraon** [vercel/next.js#17993] [Help]
Anyone knows how to solve this ? 

    "build": "NODE_ENV=production next build",
    "start": "next start -p 8080",
    "dev": "next dev",
    
    "lint": "eslint src --ext js",
    "test": "jest",
    "release": "standard-version"

**Comment by jamesmosier** [comment]
Hi @barbaraon. Are you trying to run your app in production mode? Or in some other way?

For production, are you running `npm run build` and then `npm start` to start your server after building?

**Reply by barbaraon** [reply]
Hi @jamesmosier , I am having this problem when I try to deploy in app engine. This  error ocurred inside the cloud.

**Reply by jamesmosier** [reply]
What steps are you following in order to run your Next.js application on app engine?
